test/data/javascript.js

```diff
@@ -1196,6 +1196,17 @@ exports.test_data = {
                     ')'
                 ]
             },
+            {
+                comment: "Issue 810 - es6 object literal detection",
+                unchanged: [
```

Comment:
You might as well put both inputs that you listed in the issue report in here. They were just a little more interesting than this - worth covering.